scss/main.scss

```diff
@@ -46,6 +46,12 @@ $animation-length: 80s;
   pointer-events: none !important;
 }
 
+@media only screen and (min-width: 1750px) {
+    #about{
```

Comment:
oo so since we want to move the wave we'll need to increase the margin-top of the #intro element which has the wave as the background image